### ISSUE TITLE
ProgressButton not defined, because there is no update

### ISSUE THREAD

**Issue by sebastian-marinescu** [BobRay/UpgradeMODX#61]
When just visiting the dashboard on a up-to-date MODX, 
I get the following JS-error in the console: 

```js
progressButton.js:76 Uncaught TypeError: Cannot read property 'getAttribute' of null
    at ProgressButton._validate (progressButton.js:76)
    at ProgressButton._init (progressButton.js:67)
    at new ProgressButton (progressButton.js:57)
    at ugm-bottom.js:43
``` 
The JS is right, there is no Button: 
![image](https://user-images.githubusercontent.com/3798871/51449802-0456a880-1d2e-11e9-9d31-a1a10911c575.png)

**Comment by BobRay** [comment]
Thanks for reminding me. I've seen that. It's harmless, but should be fixed. I'm thinking of just putting the same button in the "up-to-date" message but disables and with display:none, but haven't had time to try it.

**Comment by sebastian-marinescu** [comment]
Hi Bob, yeah it is harmless, but I (and probably all others, _including you_) don't like JS errors in the console :)

Why not adapt the JS and not even initialize the button, if it's not there?

**Comment by BobRay** [comment]
I agree completely and intend to fix it. It's just that I'm in the middle of about six projects right now, some of which are not running at all. ;)

**Comment by sebastian-marinescu** [comment]
Okay, thanks for the info :) 
I'll see if I can soon-ish take care of this myself 👍
